sys/dev/goldfish/goldfish_rtc.c

```diff
@@ -0,0 +1,182 @@
+/*-
+ * SPDX-License-Identifier: BSD-2-Clause-FreeBSD
+ *
+ * Copyright (c) 2020 Jessica Clarke <jrtc27@FreeBSD.org>
+ *
```

Comment:
Should this include a funding ack?

Reply:
I didn't include one for syscon_power. I'm also not really on ECATS funding (expenses go through that, but I'm not on an ECATS-funded grant).